src/js/components/Friends/AddFriendsByEmailBulk.jsx

```diff
@@ -7,167 +7,118 @@ import FriendActions from "../../actions/FriendActions";
 import FriendStore from "../../stores/FriendStore";
 import VoterStore from "../../stores/VoterStore";
 
-export default class AddFriendsByEmail extends Component {
-  static propTypes = {
-  };
+// this is currently not called by the interface
+export default class AddFriendsByEmailBulk extends Component {
+  static senderEmailAddressVerified () {
+    return true;
+  }
 
   constructor (props) {
     super(props);
     this.state = {
-      add_friends_message: "Please join me in preparing for the upcoming election.",
-      email_addresses: "",
-      email_addresses_error: false,
-      sender_email_address: "",
-      sender_email_address_error: false,
-      redirect_url_upon_save: "/friends/sign_in", // TODO DALE Remove this?
+      addFriendsMessage: "Please join me in preparing for the upcoming election.",
+      emailAddresses: "",
+      emailAddressesError: false,
+      senderEmailAddress: "",
+      senderEmailAddressError: false,
+      redirectURLUponSave: "/friends/sign_in", // TODO DALE Remove this?
       loading: false,
-      on_enter_email_addresses_step: true,
-      on_collect_email_step: false,
-      on_friend_invitations_sent_step: false,
+      onEnterEmailAddressStep: true,
```

Comment:
Hi @CoffeeWarrior, when converting the names be extra careful to not change the variable names, for example:
`on_enter_email_addresses_step` got converted to `onEnterEmailAddressStep` and it should be `onEnterEmailAddressesStep` (with "es")

`on_friend_invitations_sent_step` also got converted to `onFriendInvitationSentStep` (missing the "s" at the end of "Invitations")

I'm going to approve this, but wanted to flag this for the future.